packages/custom-eslint-rules/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "eslint-plugin-kibana-custom",
+  "version": "1.0.0",
+  "description": "Custom kibana rules",
+  "keywords": [
+    "eslint",
+    "eslintplugin",
+    "eslint-plugin"
+  ],
+  "author": "Stacey",
+  "main": "index.js",
+  "dependencies": {
+    "requireindex": "~1.1.0"
+  },
+  "devDependencies": {
+    "eslint": "~3.9.1",
+    "mocha": "^3.1.2"
+  },
+  "engines": {
+    "node": ">=0.10.0"
+  },
+  "license": "ISC"
```

Comment:
Looks like this file might be mostly copied, e.g. `requireindex`, `mocha`. I think this should be [a minimal npm package.json](https://docs.npmjs.com/getting-started/using-a-package.json#requirements), if possible, and only specify `name`, `version`, and whatever deps are actually needed (if any). 

(Could maybe keep the `description` and make it `Custom ESLint rules for Kibana`)